e2e/cypress/integration/team_settings/invite_members_spec.js

```diff
@@ -130,109 +217,8 @@ function inviteUserToTeamAsSysadmin(testTeam, user) {
 
 function closeAndComplete() {
     // # Close modal
-    cy.get('#closeIcon').should('be.visible').click();
+    cy.uiClose();
 
     // * Verify the modal closed
     cy.get('.InvitationModal').should('not.exist');
 }
-
-describe('Invite Members', () => {
```

Comment:
Rearranged so that test can be seen first before the helper functions. (Our typical code style)